README.md

```diff
@@ -1,19 +1,16 @@
-### hey, I'm Molim 👋
-
-const molim = {
+# Hey, I'm Molim 👋
 
+```js
+const Molim = {
   pronouns: "she" | "her",
-  
-  code: [Javascript, HTML, CSS, Tailwind],
```

Comment:
On line 5 you did right about string, but here you define it as reference.

When you have situations like this, always paste the code you want to show in a code editor.

If you're using a public computer and you don't have access to an editor, there's a few options:

- https://codesandbox.io/
- https://vscode.dev/
- https://stackblitz.com/